plugins/builds/README.md

```diff
@@ -0,0 +1,64 @@
+# Builds Plugin
```

Comment:
Do you think it would be helpful to include a section in the main README that points to the README for these plugins?